### ISSUE TITLE
RESP possible break change note

### ISSUE THREAD

**Issue by hwware** [madolson/placeholderkv#51]
RESP represents Redis Serialization Protocol.
In the future, maybe we should consider a new protocol to replace it, 
but the format could support back compatible with redis or it will be break change

**Comment by mattsta** [comment]
Interesting note!

Anywhere `R` is used in an acronym we could just say it now stands for `Replacement` or `Reusable` and it still works.

Also for any future API or protocol matching, copying API formats and protocols isn't restricted: https://www.jdsupra.com/legalnews/supreme-court-finds-google-s-copying-of-1447009/

**Comment by madolson** [comment]
That's true. For this thread I'm more concerned about Redis changing the RESP format (maybe doing a RESP4). I think for now the risk is really small.
